### ISSUE TITLE
Value of type '<sometype>' has no member max

### ISSUE THREAD

**Issue by Fab1n** [realm/realm-cocoa#2588]
What the heck? 
I am using a carthage build with swift 2.0 and built earlier today as Realm now supports Carthage with Swift 2 again...
I am making use of `List`'s `max` method. But it simply doesn't exist or it seems to... 

What is happening here?

I tried out the current master branch of your repo... there it works when manipulating some tests

Need help or clarification

**Comment by jpsim** [comment]
We added `max()` to `List` in #2321, which is in `master` but not in 0.95.2 (or any other release). This will be included in the next major release (0.96).

Until then, you can convert the `List` to a `Results`, which has [`max()`](https://realm.io/docs/swift/latest/api/Classes/Results.html#/s:FC10RealmSwift7Results3maxU__FGS0_Q__US_10MinMaxType__FSSGSqQ__):

```swift
let list: List<MyModel> = ...
let maxOfIntProperty = list.filter("TRUEPREDICATE").max("intProperty") as Int
```

**Comment by Fab1n** [comment]
Ok, this confused me and now I know why. I only looked at the dates of the merges and so and thought that it should be in 0.95.2
